plugins/python/python_plugin.c

```diff
@@ -242,6 +242,10 @@ int uwsgi_python_init() {
 	}
 
 	if (up.home != NULL) {
+		if(!uwsgi_is_dir(up.home)) {
+			uwsgi_log("Python Home is not a directory: %s\n", up.home);
+			return 0;
```

Comment:
I'm not sure anyone is looking at the plugin init return code, not sure if we can make this an exit(1)

Reply:
OK, I assumed since the "successful" path returned a 1, a 0 would mean fail.

Reply:
Other modules exit(1) on errors in init, so I'll follow.